Commands/rottentomatoes.py

```
@@ -1,93 +1,100 @@
 
-def rottentomatoes(query, year=None):
-    if not query:
-        return "No query provided!"
 
-    current_year = datetime.now().year
-    if year and year > current_year:
-        return "Invalid year provided!"
-
-    response = requests.post(
+def _search_rt(query):
+    r = requests.post(
```

Ruff (0.15.10)

[error] Probable use of `requests` call without timeout

(S113)